input/eBOSS_QSO_1.yml

```diff
@@ -0,0 +1,16 @@
+SDSS_name: 'QSO_1'
+num_z_bins: 50
+nside_mask: 512
+nside_nl_threshold: 4096
+lmin_nl_from_data: 2000
+data_catalogs:
+  - '/mnt/extraspace/damonge/Datasets/eBOSS/eBOSS_QSO_clustering_data-NGC-vDR16.fits'
+  - '/mnt/extraspace/damonge/Datasets/eBOSS/eBOSS_QSO_clustering_data-SGC-vDR16.fits'
+random_catalogs:
+  - '/mnt/extraspace/damonge/Datasets/eBOSS/eBOSS_QSO_clustering_random-NGC-vDR16.fits'
+  - '/mnt/extraspace/damonge/Datasets/eBOSS/eBOSS_QSO_clustering_random-SGC-vDR16.fits'
+z_edges: [1.5, 4.0]
```

Comment:
outside default z_edges in mapper

Reply:
I don't understand

Reply:
the range of the default z_edges only goes to z=3. Should we change the default in the mapper?

Reply:
But that doesn't really matter, right? This bin is meant to capture all QSOs beyond z=1.5, so this is just makes sure we get all of them.
I made it [1.5, 4] because that's actually the redshift range we used in Carlos' paper

Reply:
Ok!